### ISSUE TITLE
Make loading OpenModelica library at startup optional and allow to unload it

### ISSUE THREAD

**Issue by adrpo** [OpenModelica/OpenModelica#13331]
### Description

Currently we load the OpenModelica library in OMEdit automatically and is impossible to unload it.
We should make that configurable. 

I wanted to load my OpenModelica compiler directory named "OpenModelica" to do debugging and that is not possible.

**Comment by casella** [comment]
As I understand, we implemented this feature a long time ago. Just go to Options | Libraries, deselect Load latest Modelica version on startup and make sure it does not show up in the other two lists.

It is also possible to unload it: right click and do Unload. I do it all the time to work on the development version of MSL, because I'm too lazy to change the other configuration option.

@adrpo, do I miss something?

**Comment by adeas31** [comment]
@casella you misunderstood it with Modelica.

OpenModelica item is always shown and is fixed. We use it for autocompletion of annotations in the text view.

@adrpo already talked with me about it on slack. I will change the implementation to use some internal data structures for auto complete and will make OpenModelica optional.

**Comment by casella** [comment]
> @casella you misunderstood it with Modelica.

My bad. I was just wondering why @adrpo would make such an obvious mistake 😅 

Truth is, I'm looking through way too many tickets 😄

**Comment by adeas31** [comment]
With PR #13498 `OpenModelica` is not loaded by default at startup.

I added `OpenModelica` to the list of system libraries so user can load/unload it.

**Comment by casella** [comment]
@adeas31 I agree with @adrpo that loading OpenModelica should be configurable. However, I'm not sure that setting the default to "not load" is a good idea. Most people are not aware this library even exists, so why should they change the default?

Maybe it's better to set the default to true. @adrpo what do you think?

**Comment by casella** [comment]
You are right. I never looked into it because I think I know enough about OpenModelica already 😄. Now that I looked into it, there's probably no value for beginners in seeing that, so I'm fine with PR 13498.

BTW, I didn't know that autocompletion is based on that, good to know. As the wise Solon reportedly said: Γηράσκω δ'αεί πολλά διδασκόμενος (I'm getting older, but still learning lots of things).
